src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+            super(description);
+        }
+        public void setAt(String at ){
+            this.at = at;
+        }
+        @Override
+        public String toString(){
+            return "[E]"+super.toString()+" (at: "+this.at+")";
+        }
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+        String command;
+        Task [] tasks = new Task[100];
+        int tasksTotal = 0;
+        Task taskToAdd = null;
+        Task taskToMark;
+        Scanner in = new Scanner(System.in);
+        command = in.nextLine();
+        while(!command.equals("bye")){
+            if(command.equals("list")) printList(tasks, tasksTotal);
+            else if(command.contains("done")){
+                int index = Integer.parseInt(command.substring("done".length()+1));
+                taskToMark = tasks[index-1];
+                taskToMark.markedAsDone();
+            }
+            else if(command.contains("todo")||command.contains("deadline")||command.contains("event")){
+                if (command.contains("todo")) {
+                    String taskTodo = command.substring("todo".length()+1);
+                    taskToAdd = new Todo(taskTodo);
+                }
+                if(command.contains("deadline")){
+                    int position = command.indexOf("/");
+                    String taskDeadline = command.substring("deadline".length()+1,position-1);
+                    String by = command.substring(position+"/by".length()+1);
+                    taskToAdd = new Deadline(taskDeadline);
+                    ((Deadline) taskToAdd).setBy(by);
+                }
+                if(command.contains("event")){
+                    int position = command.indexOf("/");
+                    String taskEvent = command.substring("event".length()+1,position-1);
+                    String at = command.substring(position+"/at".length()+1);
+                    taskToAdd = new Event(taskEvent);
+                    ((Event) taskToAdd).setAt(at);
+                }
+                tasks[tasksTotal]=taskToAdd;
+                tasksTotal++;
+                printAddMessage(taskToAdd);
+                printNumOfTasksInList(tasksTotal);
+            }
+            command = in.nextLine();
+        }
+        printExitMessage();
+
+
+    }
+    public static void printExitMessage(){
+        printLine();
+        System.out.println("     Bye. Hope to see you again soon!");
+        printLine();
+    }
+    public static void printWelcomeMessage(){
+        printLine();
+        System.out.println("     Hello！ I'm Duke");
+        System.out.println("     What can I do for you?");
+        printLine();
+    }
```

Comment:
I like how you combine the greeting messages into one function printWelcomeMessage(). Keep it up.